src/test-qualification-workspace/test-qualification-workspace.service.ts

```diff
@@ -128,6 +139,8 @@ export class TestQualificationWorkspaceService {
     }
 
     record.testClaimCode = payload.testClaimCode;
+    // Set the TestClaimCode relation to null to avoid TypeScript error
+    record.TestClaimCode = null;
```

Comment:
I don't see any TypeScript error when removing this, and adding it may cause a DB error.
```suggestion
```